src/ColorPicker/ColorPicker.scss

```diff
@@ -0,0 +1,40 @@
+@import "variables";
+
+.pgn__color-picker {
+  border: none;
+  background:
+    linear-gradient(to bottom right,
+    rgb(255, 0, 0) 0%,
+    rgb(255, 154, 0) 17%,
+    rgb(208, 222, 33) 33%,
+    rgb(79, 220, 74) 40%,
+    rgb(28, 127, 238) 75%,
+    rgb(95, 21, 242) 86%,
+    rgb(186, 12, 248) 100%
```

Comment:
Are you going to put a color picker icon in here?

Reply:
So I talked to design and decided against it because the eyedropper has its own implied functionality (of being able to select colors on the page) which we don't have the option for.